packages/kernel/src/kernels.ts

```diff
@@ -14,6 +14,11 @@ import { Mutex } from 'async-mutex';
 
 import { PageConfig } from '@jupyterlab/coreutils';
 
+/**
+ * Use the default kernel wire protocol.
```

Comment:
Perhaps pull this from [upstream](https://github.com/jupyterlab/jupyterlab/blob/9be8052d846290e22a0e51a5d3e3dec43ba9efdc/packages/services/src/kernel/messages.ts#L1268)?

Reply:
It's not exported at the `@jupyterlab/services` level but we can indeed import it from `@jupyterlab/services/lib/kernel/messages`.